### DISCUSSION TITLE
About creating a plugin https://tvplus.com.tr/ plugin

### DISCUSSION THREAD

**Discussion by omergumus** [streamlink/streamlink#4749] [General]
Is it possible to create a plugin for this platform that offers Turkish broadcasts and content?

source link: https://tvplus.com.tr/ plugin 

About creating a plugin

**Comment by bastimeyer** [comment]
Read https://github.com/streamlink/streamlink/discussions/4578

The discussion forum is not meant for plugin requests. Create a proper plugin request on the issue tracker and fill in all of the required details. Locking the thread...
